charts/logging-operator-fluent/templates/fluentd-cr.yaml

```diff
@@ -19,9 +19,10 @@ spec:
   configReloaderImage: {{ toYaml .Values.fluentd.configReloaderImage | nindent 4 }}
   resources: {}
   fluentdPvcSpec: {{ toYaml .Values.fluentd.fluentdPvcSpec | nindent 4 }}
+  tolerations: {{ toYaml .Values.fluentd.tolerations | nindent 4 }}
```

Comment:
I would ask to put this in an if statement to leave the tolerations off if not defined